R/layers-features.R

```diff
@@ -0,0 +1,34 @@
+#' Constructs a DenseFeatures.
+#' 
+#' A layer that produces a dense Tensor based on given feature_columns.
+#' 
+#' @inheritParams layer_dense
+#' 
+#' @param feature_columns An iterable containing the FeatureColumns to use as 
+#'    inputs to your model. All items should be instances of classes derived from 
+#'    `DenseColumn` such as `numeric_column`, `embedding_column`, `bucketized_column`, 
+#'    `indicator_column`. If you have categorical features, you can wrap them with an 
+#'    `embedding_column` or `indicator_column`. See `tfestimators::feature_columns()`.
+#'   
+#' @family core layers   
+#'   
+#' @export
+layer_dense_features <- function(object, feature_columns, name = NULL,
+                                 trainable = NULL, input_shape = NULL,
+                                 batch_input_shape = NULL, batch_size = NULL, dtype = NULL, 
+                                 weights = NULL) {
+
+  if (!is_tensorflow_implementation() || !tensorflow::tf_version() >= "2.0")
+    stop("layer_dense_features requires TensorFlow implementation and version >= 2.0")
+  
+  create_layer(keras$api$`_v2`$keras$layers$DenseFeatures, object, list(
```

Comment:
DenseFeatures is actually implemented in the tf.feature_columns module and aliased to keras within the api. Not sure this is the right way to use it though.

Reply:
This feels reasonable to me.